i18n/dummy.py

```diff
@@ -138,18 +138,24 @@ class Dummy(BaseDummyConverter):
         """.split()
     )
 
-    # To simulate more verbose languages (like German), pad the length of a string
-    # by a multiple of PAD_FACTOR
-    PAD_FACTOR = 1.33
-
     def pad(self, string):
-        """add some lorem ipsum text to the end of string"""
+        """
+        Add some lorem ipsum text to the end of string to simulate more verbose languages (like German).
+        Padding factor determined by http://www.w3.org/International/articles/article-text-size.en
+        """
         size = len(string)
-        if size < 7:
+        if size <= 10:
             target = size * 3
+        elif size <= 20:
+            target = size * 2
```

Comment:
The odd thing here is that the result is non-monotonic: 10 becomes 30, but 11 becomes 22.  Not to get too dorky about it, but this fits the data pretty well:  `target = size * (4.75 - size ** 0.27)`

Another option is to do it like taxes: the first 10 chars *= 3, then the chars between 10 and 20 *= 2, etc.

Also, not sure if these tweaks are worth it :)